mmengine/visualization/visualizer.py

```diff
@@ -441,6 +444,12 @@ def draw_texts(
                 the texts will have the same font family.
                 font_familiy can be 'serif', 'sans-serif', 'cursive', 'fantasy'
                 or 'monospace'.  Defaults to 'sans-serif'.
+            font_properties (Union[FontProperties, List[FontProperties]],
```

Comment:
We can add docstring of the recommended setting value of this parameter in the case of Chinese text here.  Otherwise, users still don't know how to set this parameter.